src/py/flwr/server/superlink/state/sqlite_state.py

```diff
@@ -54,7 +54,7 @@
     consumer_node_id        INTEGER,
     created_at              TEXT,
     delivered_at            TEXT,
-    ttl                     TEXT,
+    ttl                     FLOAT,
```

Comment:
According to https://www.sqlite.org/datatype3.html, I think it would be better to use `REAL` instead of `FLOAT` (which will be interpreted as `REAL` under the hood.)

Reply:
i did this. thanks!